src-11ty/redirect-page.html

```diff
@@ -10,6 +10,9 @@
 <style>
   .ha-card {
     min-height: 203px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
```

Comment:
```suggestion
```

This makes the placeholder jump from bottom to top when the bottom-bar is drawn. I think that we should remove it